cmd/zed/index/create.go

```diff
@@ -20,9 +20,23 @@ import (
 
 var create = &charm.Spec{
 	Name:  "create",
-	Usage: "create [options] rule-name pattern",
+	Usage: "create [options] rule-name rule-type value",
 	Short: "create an index rule for a lake",
-	New:   newCreate,
+	Long: `
+The index create command creates an index rule that can be applied to any
+pool in the Zed lake. The command takes three arguments: the name of the rule,
```

Comment:
Most other `Long` values use two spaces after a sentence.
```suggestion
pool in the Zed lake.  The command takes three arguments: the name of the rule,
```